autoload/arduino.vim

```diff
@@ -367,7 +426,7 @@ function! arduino#Verify() abort
     call slime#send(cmd."\r")
   elseif !empty($TMUX) && !empty(g:arduino_verify_tmux)
     " the call to $SHELL -i is to prevent the split to directly close after the command has finished
```

Comment:
This comment is apparently out-of-date. We can just delete it and all the copies

Reply:
Ok